src/amo/reducers/versions.js

```diff
@@ -106,7 +112,13 @@ export const getVersionsBySlug = ({
   invariant(state, 'state is required');
 
   const infoForSlug = state.bySlug[slug];
-  return (infoForSlug && infoForSlug.versions) || null;
+  const versionIds = infoForSlug && infoForSlug.versionIds;
+  if (versionIds) {
+    return versionIds.map((versionId) => {
+      return state.byId[versionId];
```

Comment:
This should throw an error if the version object doesn't exist in state. It would mean the app is severely broken. If you let it return `undefined` (or `null`) then it might create an infinite loading loop depending on the loading code.